apps/www/_blog/2023-04-01-supaclub.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: SupaClub
+description: The Worlds First Software Engineering Nightclub.
+author: ant_wilson
+image: supaclub/supaclub-thumb.png
+thumb: supaclub/supaclub-thumb.png
+tags:
+  - announcements
+date: '2023-04-01'
+toc_depth: 3
+---
+
+Announcing SupaClub (pronounced “Super-Clurb”), the world’s first Software Engineering Nightclub.
+
+Located on the Indonesian island of Bali, SupaClub is where the digital nomads of South East Asia are mixing Hardcore Techno with Hardcore Engineering.
+
+![Main Dance Floor](/images/blog/supaclub/supaclub1.png)
+
+In March this year, some members of the Supabase team took advantage of the fact that they can work fully remotely, and spent a week working in Bali. The island is a great place to relax in coffee shops during the day, but when night time rolled around our team faced a major problem - the large established nightclubs of Bali would not allow our engineers to bring their laptops into the club. They have strict rules against EMACS on the dance floor, killing our dream of fixing bugs whilst jamming to 90s club anthems.
+
+Our team got together and decided that we would do what we do best at Supabase. We’d ship a fix, and today that fix becomes a reality.
+
+![coding in the club](/images/blog/supaclub/supaclub2.png)
+
+SupaClub, situated near the beach in the Canggu neighborhood of Bali, will open its doors this coming Friday. It can house 4000 developers across 4 floors, each with a dedicated DJ and breakout rooms. It has UltraSpeed 1Gbps WiFi and bookable standing desks on every floor. VIP [SupaBrew bottle service](https://supabase.com/blog/supabrew) is available inside private conference rooms, complete with whiteboards and UltraWide monitors for full immersion (Think WeWork meets Pacha).
+
+![dancefloor meets coding](/images/blog/supaclub/supaclub3.png)
+
+In terms of beverage selection, [SupaBrew](https://supabase.com/blog/supabrew) will of course be available behind the bar, along with a cocktail list that includes:
+
+- **Bjarne Pisco Stroustrup** - A back-to-basics tantalizing mix of tart, sweet, and silky pisco brandy that won't leave your pointers dangling
+- **The Linus** - A spicy twist on the Bloody Mary, this aggressive blend will have you shouting for more
+- **Mojito-js** - we would list the ingredients but our mixologists keep changing the spec
+- **HTML Highball** - Not a real cocktail, more of a mocktail
+
+SupaClub was made possible with the help of infamous Fyre Fest investor Ja Rule and “Super Bass” songstress Niki Minaj, both of which will also be performing on the opening night.
+
+If you would like to reserve a spot on the guest list for the opening night please email us at noreply@supabase.com with a link to your GitHub account and "Re: april fools" in the subject line. See you there.
```

Comment:
Do we want to finish with the fact that this is an April fools joke, but we did actually go to bali? (and could talk about our offsite a bit)